src/metatensor/models/cli/conf/architecture/soap_bpnn.yaml

```diff
@@ -3,30 +3,30 @@ name: soap_bpnn
 
 model:
   soap:
-    cutoff: 5.0
-    max_radial: 8
-    max_angular: 6
-    atomic_gaussian_width: 0.3
+    cutoff: 5.0  # radial cutoff (Å)
+    max_radial: 8  # number of radial basis functions
+    max_angular: 6  # maximum degree of spherical harmonics
+    atomic_gaussian_width: 0.3  # width of atomic Gaussian (Å)
     radial_basis:
       Gto: {}
-    center_atom_weight: 1.0
+    center_atom_weight: 1.0  # weight of center atom in SOAP
     cutoff_function:
       ShiftedCosine:
-        width: 1.0
+        width: 1.0  # width of shifted cosine cutoff function (Å)
     radial_scaling:
       Willatt2018:
```

Comment:
I think we should also add a comment on this.

Reply:
maybe instead of re-documenting everything, this could be a link to https://luthaf.fr/rascaline/latest/references/calculators/soap-power-spectrum.html?